logFileReplay-csv/send_csv_data_py3.py

```diff
@@ -161,10 +161,13 @@ def send_data_to_receiver(post_url, to_send_data, num_of_message):
         for row in reader:
             entry = {}
             entry['tag'] = row[csv_vars['instance_field']]
-            timestamp = arrow.get(row[csv_vars['timestamp_field']], csv_vars['timestamp_format'], tzinfo=csv_vars['timestamp_timezone'])
-            # convert timezone to utc required by api 
-            timestamp = timestamp.to(pytz.utc)
-            entry['eventId'] = timestamp.timestamp() * 1000
+            if len(timestamp_format) == 0:
+                entry['eventId'] = int(row[csv_vars['timestamp_field']]) * 1000
```

Comment:
This assumes the timestamp field is an integer that denotes the EPOCH time in seconds. 

Instead we should use arrow.get to parse the integer and then make sure that we get the timestamp.timestamp() value which should be the epoch time in seconds.